src/main/java/de/retest/recheck/ui/diff/ElementIdentificationWarning.java

```diff
@@ -19,14 +19,22 @@
 	private static final long serialVersionUID = 1L;
 
 	@XmlAttribute
-	private final String testClassName;
+	private final String testFileName;
 
 	@XmlAttribute
 	private final Integer testLineNumber;
 
+	@XmlAttribute
+	private final String findByMethodName;
+
+	@XmlAttribute
+	private final String qualifiedTestName;
```

Comment:
What is the difference between `testFileName` and this? For what do we need which information?

Reply:
When implementing in the downstream projects, I realized that there can be test classes that life in files of which the names differ. So you could (in theory) have a TestXY that lives in a de.retest.Tests class. Also, you could have multiple Tests.java files, that differ in package/path. Will document via JavaDoc.

Reply:
So, `testFileName` is just the name of the source file (e.g. `MyTest` or `MyIT`), whereas `qualifiedTestName` is the name of the fully-qualified test class name (e.g. `com.example.MyTest` or `com.example.MyTest.MyNestedTest`)?

Reply:
Ah, I see. 👍 As you said, some minimal Javadoc would be helpful, maybe with an actual example.